### PR TITLE
[OSDOCS#9302]: RN for OSASINFRA-3207 (Integrate CAPO into cluster-capi-operator (TP))

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-9302

Link to docs preview:
https://70182--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-installation-and-update

QE review:
N/A

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -58,6 +58,8 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-installation-and-update"]
 === Installation and update
 
+The Cluster API Provider for OpenStack (CAPO) is now deployed by default when the `TechPreviewNoUpgrade` feature flag is enabled. The lifecycle of this component is managed by the CAPI cluster Operator. When the `TechPreviewNoUpgrade` feature flag is enabled, an OpenStack cluster and additional OpenShift cluster are automatically created for the current OpenShift cluster. It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Mail Application Programming Interface (MAPI) resources are configured. It is important to note that these resources are equivalent but not identical.
```
Comment (jab-rh) [inline release_notes/ocp-4-15-release-notes.adoc:61]:
```suggestion
The Cluster API Provider for OpenStack (CAPO) is now deployed by default when the `TechPreviewNoUpgrade` feature flag is enabled. The lifecycle of this component is managed by the CAPI Operator. When the `TechPreviewNoUpgrade` feature flag is enabled, an OpenStack cluster and additional OpenShift cluster are automatically created for the current OpenShift cluster. It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Mail Application Programming Interface (MAPI) resources are configured. It is important to note that these resources are equivalent but not identical.
```

**Review by jab-rh — COMMENTED** [review]
@obrown1205, left a comment. We've always just called them _Operators_, but if that's different in this instance, feel free to disregard. Thanks!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (obrown1205) [reply release_notes/ocp-4-15-release-notes.adoc]: done :)

**Comment by mjpytlak** [comment]
@obrown1205 I was resolving a merge conflict when I noticed that this entry does not have a heading. Was there a specific reason why you did not add a heading or is this just an oversight?

**Comment by obrown1205** [comment]
@mjpytlak Shoot - good catch. I'll create a small PR now to address it. :)

**Pull request merged**